docs/built-in-commands.md

```diff
@@ -0,0 +1,30 @@
+---
+id: built-in-commands
+title: "Built-in commands"
+---
+**ZIO CLI** automatically constructs some common commands:
+- `--help` or `-h` shows a description of the CLI's commands
+- `--wizard` executes wizard mode
+- `--shell-completion-index`, `--shell-completion-script` and `--shell-type` are used for Bash and Zsh Completion.
+
+These commands are automatically generated also for all subcommands of a command.
```

Comment:
Good to have a section on each command, and show the real output from it.

In general, for any CLI App, you should always show the man page to give user's a feel for what the docs are that they get generated automatically.